### PR TITLE
Fix third party integration tests

### PR DESCRIPTION
**Pull request by alexsu52**
#third_party_sanity build 24

### CODE REVIEW

File: tests/torch/test_sanity_third_party.py [hunk 1]
```diff
@@ -260,9 +262,10 @@ def test_install_mmdet(self, pip_cache_dir):
         pip_runner.run_pip("install mmcv-full==1.2.0 "
                            "-f https://download.openmmlab.com/mmcv/dist/cu102/torch{}/index.html".format(
             BKC_TORCH_VERSION),
-                           cwd=self.MMDET_PATH)
+                           cwd=self.MMDET_PATH,
+                           use_cache=False)
         pip_runner.run_pip("install onnx onnxruntime", cwd=self.MMDET_PATH)
-        pip_runner.run_pip("install torchvision", cwd=self.MMDET_PATH)
+        pip_runner.run_pip("install torchvision==0.9.1", cwd=self.MMDET_PATH)
```
Comment (vshampor) [inline tests/torch/test_sanity_third_party.py:268]: Should we set up a constant for this in nncf/version.py, along with `BKC_TORCH_VERSION`?

File: tests/torch/test_sanity_third_party.py [hunk 2]
```diff
@@ -233,7 +235,7 @@ def setup(self, temp_folder):
         self.PATH_TO_PATCH = str(os.path.join(PROJECT_ROOT, "third_party_integration", "mmdetection",
                                               "0001-Modifications-for-NNCF-usage.patch"))
         self.VENV_MMDET_PATH = str(os.path.join(temp_folder["venv"], "mmdet"))
-        self.activate_venv = str(". {}/bin/activate".format(self.VENV_MMDET_PATH))
+        self.activate_venv = str("CUDA_HOME=/usr/local/cuda-10.2 . {}/bin/activate".format(self.VENV_MMDET_PATH))
```
Comment (vshampor) [inline tests/torch/test_sanity_third_party.py:238]: Should this be done on the docker container level instead?

Comment (alexsu52) [reply tests/torch/test_sanity_third_party.py:238]: @pfinashx Could you fix cuda version in the docker container for the third-party integration test?

File: tests/torch/test_sanity_third_party.py (hunk 1 above)
Comment (alexsu52) [reply tests/torch/test_sanity_third_party.py]: Done

**Review by vshampor — APPROVED** [review]

**Pull request merged**